source/client-integration/create-client-configuration.rst

```diff
@@ -52,10 +52,16 @@ A client configuration includes all organization specific configuration and all
 
             const string organizationNumber = "123456789";
 
+            const string proxyAddress = "http://proxyserver:80/";
+            const string userName = "username";
+            const string userPassword = "password";
+
             var clientConfiguration = new ClientConfiguration(
                 Environment.DifiTest,
                 CertificateReader.ReadCertificate(),
-                new Sender(organizationNumber)
+                new Sender(organizationNumber),
+                new WebProxy( proxyAddress ,true),
```

Comment:
Trenger man å bypass'e proxy for lokale adresser? Klienten skal vel i utgangspunktet ikke koble til noe lokalt?
```suggestion
                new WebProxy(proxyAddress),
```

Reply:
resolved